Exercise/src/tests.cpp

```diff
@@ -0,0 +1,48 @@
+#include <gtest/gtest.h>
+
+#include "my_string.h"
+class CMyStringFixture : public ::testing::Test
+{
+protected:
+	Exercise_1::CMyString cmsMyString;
+	const size_t sTestDataLength{17};
+	const char cTestData[17]{"Some test string"};
```

Comment:
avoid some "magic numbers" like 17
It is ok it tests, but still...

Reply:
Done nothing because those are tests